test/components/reading-plan.spec.coffee

```diff
@@ -0,0 +1,81 @@
+{expect} = require 'chai'
+
+React = require 'react'
+
+{TaskPlanActions, TaskPlanStore} = require '../../src/flux/task-plan'
+{TocActions} = require '../../src/flux/toc'
+{ReadingPlan} = require '../../src/components/task-plan/reading'
```

Comment:
Ah, the reason I split `ReadingPlan` into `ReadingShell` and `ReadingPlan` was so that I could test just the Plan part. Adding the code to mock the ReactRouter would have been non-trivial and since most of the testing was on the interactions rather than how the id was loaded I opted for splitting the file up